### PR TITLE
fixed chap 14.2/3 wording

### PR DESCRIPTION
**Pull request by zhunation**

### CODE REVIEW

File: ch14/bias_modeling.md [hunk 1]
```diff
@@ -1 +1 @@
-!INCLUDE "../notebooks-html/bias_modeling.html"
+!INCLUDE "../notebooks-html\bias_modeling.html"
```
Comment (SamLau95) [inline ch14/bias_modeling.md]: Hmm. It looks like the reverse backslashes `\` should be regular slashes `/`. This looks like a issue with running the conversion scripts on Windows. For now, can you replace the `\` with `/` for this `.md` file?

File: ch14/bias_risk.md [hunk 2]
```diff
@@ -1 +1 @@
-!INCLUDE "../notebooks-html/bias_risk.html"
+!INCLUDE "../notebooks-html\bias_risk.html"
```
Comment (SamLau95) [inline ch14/bias_risk.md]: This one too.

File: notebooks-html/bias_modeling.html [hunk 3]
```diff
@@ -77,38 +77,39 @@
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
 <h2 id="Model-Bias-and-Variance">Model Bias and Variance<a class="anchor-link" href="#Model-Bias-and-Variance">&#182;</a></h2><p>We have previously seen that our choice of model has two basic sources of error.</p>
-<p>Our model may be too simple—a linear model is not able to properly fit data generated from a quadratic process, for example. This type of error arises from model <strong>bias</strong>.</p>
-<p>Our model may also fit the random noise present in the data—even if we fit a quadratic process using a quadratic model, the model may predict different outcomes than the true process produces. This type of error arises from model <strong>variance</strong>.</p></div></div></div>
+<p>Our model may be too simple�a linear model is not able to properly fit data generated from a quadratic process, for example. This type of error arises from model <strong>bias</strong>.</p>
```
Comment (SamLau95) [inline notebooks-html/bias_modeling.html]: Looks like the dash got removed here. Can you add it back?

File: notebooks-html/bias_modeling.html [hunk 4]
```diff
@@ -77,38 +77,39 @@
...
+<p>Our model may also fit the random noise present in the data�even if we fit a quadratic process using a quadratic model, the model may predict different outcomes than the true process produces. This type of error arises from model <strong>variance</strong>.</p></div></div></div>
```
Comment (SamLau95) [inline notebooks-html/bias_modeling.html]: Here too.

File: notebooks-html/bias_modeling.html [hunk 5]
```diff
@@ -77,38 +77,39 @@
...
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<h2 id="The-Bias-Variance-Decomposition">The Bias-Variance Decomposition<a class="anchor-link" href="#The-Bias-Variance-Decomposition">&#182;</a></h2><p>We can make the statements above more precise by decomposing our formula for model risk. Recall that the <strong>risk</strong> for a model $ f_\hat{\theta} $ is the expected loss for all possible sets of training data $ X $, $ y $ and all input-output points $ z$, $ \gamma $ in the population:</p>
-$$
+<h2 id="The-Bias-Variance-Decomposition">The Bias-Variance Decomposition<a class="anchor-link" href="#The-Bias-Variance-Decomposition">&#182;</a></h2><p>We can make the statements above more precise by decomposing our formula for model risk. Recall that the <strong>risk</strong> for a model $ f_\hat{\theta} $, is the expected loss for all possible sets of training data $ X $, $ y $ and all input-output points $ z$, $ \gamma $ in the population:</p>
```
Comment (SamLau95) [inline notebooks-html/bias_modeling.html]: There shouldn't be a comma here.

File: notebooks-html/bias_modeling.html [hunk 6]
```diff
@@ -77,38 +77,39 @@
...
-<p>The second term in this expression, $ \text{Var}(f_\hat{\theta}(z)) $, represents the model variance. The variance is low when the model's predictions don't change much when the model is trained on different datasets from the population. The variance in high when the model's predictions change greatly when the model is trained on different datasets from the population.</p>
+<p>The first term in this expression, $ (\mathbb{E}[f_\hat{\theta}(z)] - f_\theta(z))^2 $, is a mathematical expression for the squared bias, $\text{bias}^2$, of the model. The bias is equal to zero if in the long run our choice of model $ f_\hat{\theta}(z) $ predicts the same outcomes produced by the population process $ f_\theta(z) $. The bias is high if our choice of model makes poor predictions of the population process even when we have the entire population as our dataset.</p>
```
Comment (SamLau95) [inline notebooks-html/bias_modeling.html]: On a second read, I remember not stating that this term was the squared bias because I didn't think it was an important point and might be a bit confusing for people seeing bias for the first time. Do you think you can reword this to something like:

> is a mathematical expression for the bias of the model. (Technically, this term represents the squared bias.)

File: notebooks-html/bias_risk.html [hunk 7]
```diff
@@ -77,76 +77,83 @@
...
 &amp;= \frac{1}{n} \sum_{i}(y_i - f_\hat{\theta} (X_i))^2\\
 \end{aligned}
-$$<p>As before, we use $ \hat{\theta} $ as our vector of model parameters, $ x $ as a vector containing a row of a data matrix $ X $, and $ y $ as our vector of observed values to predict.</p></div></div></div>
+$$</p>
+<p>As before, we use $ \hat{\theta} $ as our vector of model parameters, $ x $ as a vector containing a row of a data matrix $ X $, and $ y $ as our vector of observed values to predict, where $ X_i $ is the ith row of $ X $ and $ y_i $ is the ith entry of y.</p></div></div></div>
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<p>Observe that our cost function is the average of the loss function values for each row of our data. If we define the loss function:</p>
-$$
+<p>Observe that our cost function is the average of the loss function values for each row of our data. If we define the loss function (squared loss):</p>
```
Comment (SamLau95) [inline notebooks-html/bias_risk.html]: Maybe we can remove the parentheses and just say:

> If we define the squared error loss function:

File: notebooks-html/bias_risk.html [hunk 8]
```diff
@@ -77,76 +77,83 @@
...
 &amp;= (y_i - f_\hat{\theta} (x))^2
 \end{aligned}
-$$<p>Then we may rewrite our cost function more simply:</p>
-$$
+$$</p>
+<p>Then we may rewrite our cost function more simply:</p>
+<p>$$
 \begin{aligned}
-L(\hat{\theta}, X, y)
+L(\hat{\theta}, X, y_i)
```
Comment (SamLau95) [inline notebooks-html/bias_risk.html]: I agree with your loss function change above, but the cost function definition should take in `y` as a parameter, not `y_i`. Notice the summation over all components of `y`—this is why we need the entire `y` vector.

File: notebooks-html/bias_risk.html [hunk 9]
```diff
@@ -77,76 +77,83 @@
...
+$$</p>
+<p>Notice that this loss is a random variable; the loss changes for different sets of observed data $ X $ and $ y $ and different points $z, \gamma $ from our population.</p></div></div></div>
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
 <p>The <strong>risk</strong> for a model $ f_\hat{\theta} $ is the expected value of the loss above for all training data $ X $, $ y $ and all points $ z$, $ \gamma $ in the population:</p>
-$$
+<p>$$
 \begin{aligned}
-R(f_\hat{\theta}) = \mathbb{E}[ \ell(\gamma, f_\hat{\theta} (z)) ]
+R(f_\hat{\theta}(x)) = \mathbb{E}[ \ell(\gamma, f_\hat{\theta} (z)) ]
```
Comment (SamLau95) [inline notebooks-html/bias_risk.html]: Oops, thanks!

File: notebooks-html/bias_risk.html [hunk 10]
```diff
@@ -77,76 +77,83 @@
...
+$$</p>
+<p>And the cost function:</p>
+<p>$$
 \begin{aligned}
 L(\hat{\theta}, X, y)
 &amp;= \frac{1}{n} \sum_{i}(y_i - f_\hat{\theta} (X_i))^2\\
 \end{aligned}
-$$<p>As before, we use $ \hat{\theta} $ as our vector of model parameters, $ x $ as a vector containing a row of a data matrix $ X $, and $ y $ as our vector of observed values to predict.</p></div></div></div>
+$$</p>
+<p>As before, we use $ \hat{\theta} $ as our vector of model parameters, $ x $ as a vector containing a row of a data matrix $ X $, and $ y $ as our vector of observed values to predict, where $ X_i $ is the ith row of $ X $ and $ y_i $ is the ith entry of y.</p></div></div></div>
```
Comment (SamLau95) [inline notebooks-html/bias_risk.html]: This sentence is long. Can we instead write split it up into two sentences?

> to predict. $ X_i $ is the $i$'th row of $ X $ and $ y_i $ is the $i$'th entry of y.

**Comment by SamLau95** [comment]
Can you also pull from master? I have an update to the build script that will generate markdown files which will make reviewing a lot easier for me.

**Comment by zhunation** [comment]
Fixed all the changes sam

File: notebooks/ch14/bias_modeling.md [hunk 11]
```diff
@@ -75,9 +75,9 @@ R(f_\hat{\theta}) = (\mathbb{E}[f_\hat{\theta}(z)] - f_\theta(z))^2 + \text{Var}
 \end{aligned}
 $$
 
-The first term in this expression, $ (\mathbb{E}[f_\hat{\theta}(z)] - f_\theta(z))^2 $, is a mathematical expression for the bias of the model. The bias is equal to zero if in the long run our choice of model $ f_\hat{\theta}(z) $ predicts the same outcomes produced by the population process $ f_\theta(z) $. The bias is high if our choice of model makes poor predictions of the population process even when we have the entire population as our dataset.
+The first term in this expression, $ (\mathbb{E}[f_\hat{\theta}(z)] - f_\theta(z))^2 $, is a mathematical expression for the bias of the model. (Technically, this term represents the biased squared, $\text{bias}^2$). The bias is equal to zero if in the long run our choice of model $ f_\hat{\theta}(z) $ predicts the same outcomes produced by the population process $ f_\theta(z) $. The bias is high if our choice of model makes poor predictions of the population process even when we have the entire population as our dataset.
```
Comment (SamLau95) [inline notebooks/ch14/bias_modeling.md]: nit: `(Technically, this term represents the biased squared, $\text{bias}^2$).` should be `(Technically, this term represents the biased squared, $\text{bias}^2$.)`. The period should be in the parentheses when the parentheses contains an entire sentence.

File: notebooks/ch14/bias_modeling.md [hunk 12]
```diff
@@ -136,7 +136,7 @@ plt.scatter(xs, ys, s=10)
 plt.title('One set of observed data');
 ```
 
-Suppose we draw many sets of data from the population and fit a linear model to each one. Below, we plot the population data generation scheme in blue and the model predictions in green.
+Suppose we draw many sets of data from the population and fit a simple linear model, to each one. Below, we plot the population data generation scheme in blue and the model predictions in green.
```
Comment (SamLau95) [inline notebooks/ch14/bias_modeling.md]: nit: There shouldn't be a comma before `to each one`.
